apps/graphql/src/apps/booking/types/outputs/ConfirmPayment.js

```diff
@@ -0,0 +1,20 @@
+// @flow
+
+import { GraphQLObjectType, GraphQLInt, GraphQLFloat } from 'graphql';
+
+const ConfirmPaymentOutput = new GraphQLObjectType({
+  name: 'ConfirmPaymentOutput',
+  fields: {
+    status: {
+      description:
+        'Should be 0 (zero) that means the confirm_payment was successful, any other result means that something went wrong',
+      type: GraphQLInt,
```

Comment:
Is there a meaning behind the other status, or could we just as well return isSuccess: Boolean here?

Reply:
I don't know, according to the documentation... 

<img width="580" alt="Screenshot 2019-04-12 at 14 58 01" src="https://user-images.githubusercontent.com/11021461/56038918-67574a80-5d33-11e9-8245-c8cce4b9c275.png">

tbf it would be better to just have a boolean :/

Reply:
So, let's return a boolean instead of this cryptic number 😊